src/components/Form/FormHelpButton.tsx

```diff
@@ -12,13 +10,21 @@ function safeId(id: string) {
   return id.replace('.', '-')
 }
 
+export interface HelpProps {
+  label: string
+  text: string
+}
```

Comment:
This interface is passed into the 'help' attribute for elements that need a help button. Having `label` in this interface removes the need to try and extract label text from the the 'label' attribute on the element.